tests/garage/tf/misc/test_tensor_utils.py

```diff
@@ -35,6 +38,22 @@ def test_compute_advantages(self):
 
         assert np.array_equal(adv, desired_val)
 
+    def test_normalize_pixel_observation(self):
```

Comment:
you will also need to test that CategoricalCNNPolicy is doing the right think here...